src/liblzma/check/crc32_loongarch.h

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: 0BSD
+
+///////////////////////////////////////////////////////////////////////////////
+//
+/// \file       crc32_loongarch.h
+/// \brief      CRC32 calculation with LoongArch optimization
+//
+//  Authors:    Xi Ruoyao
+//
+///////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef LZMA_CRC32_LOONGARCH_H
+#define LZMA_CRC32_LOONGARCH_H
+
+#include <larchintrin.h>
+#include <sys/auxv.h>
+
+// Defined in section 9.1 of
+// "Software Development and Build Convention for LoongArch Architectures"
+// (https://github.com/loongson/la-softdev-convention/).  Hard code it
+// instead of including <bits/hwcap.h> for compatibility with Glibc <= 2.37.
+#define HWCAP_LOONGARCH_CRC32 (1 << 6)
```

Comment:
What is the name defined in `<sys/auxv.h>`? Just checking to be sure it doesn't conflict with this name. It would be better to only use the hardcoded value if the correct one from `<sys/auxv.h>` is not defined.

Reply:
`<sys/auxv.h>` does not define `HWCAP_LOONGARCH_*`, it only defines `getauxval()`.

If you still think we should add an `#ifdef` here in case `HWCAP_LOONGARCH_*` is added into `auxv.h` in the future I can add it.